Engine/DataFeeds/Queues/FakeDataQueue.cs

```diff
@@ -120,6 +120,19 @@ public void Unsubscribe(LiveNodePacket job, IEnumerable<Symbol> symbols)
             }
         }
 
+        /// <summary>
+        /// Returns whether the data provider is connected
+        /// </summary>
+        /// <returns>true if the data provider is connected</returns>
+        public bool IsConnected => true;
+
+        /// <summary>
+        /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
+        /// </summary>
+        public void Dispose()
+        {
+        }
```

Comment:
nit - FakeDataQueue actually has a `Timer` to dispose of

Reply:
Will update